application/controllers/HostsController.php

```diff
@@ -50,15 +50,19 @@ public function indexAction()
 
         $limitControl = $this->createLimitControl();
         $paginationControl = $this->createPaginationControl($hosts);
+
+        $defaultSort = 'host.state.severity desc,host.state.last_state_change desc';
```

Comment:
```suggestion
        $defaultSort = ['host.state.severity DESC', 'host.state.last_state_change DESC'];
```

This looks more readable to me.